app/src/main/java/net/gsantner/markor/format/asciidoc/AsciidocTextConverter.java

```diff
@@ -0,0 +1,58 @@
+/*#######################################################
+ *
+ *   Maintained by Gregor Santner, 2018-
+ *   https://gsantner.net/
+ *
+ *   License of this file: Apache 2.0 (Commercial upon request)
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+#########################################################*/
+package net.gsantner.markor.format.asciidoc;
+
+import android.content.Context;
+
+import net.gsantner.markor.format.TextConverterBase;
+import net.gsantner.opoc.util.GsFileUtils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@SuppressWarnings("WeakerAccess")
+public class AsciidocTextConverter extends TextConverterBase {
```

Comment:
in this case you can extend from PlaintextConverter and remove convertMarkup & getContentType overrides.

Then text gets rendered in preformatted (`<pre>`) plaintext in view-mode (currently: displayed in html)

Reply:
I don't understand the comment and what should be done.

Reply:
class asciitextconverter extends plaintextconverter, remove said two methods.